other/add_default_resources/kyverno-test.yaml

```diff
@@ -0,0 +1,14 @@
+name: add-default-resources
+policies:
+  - add-default-resources.yaml
+resources:
+  - resource.yaml
+variables: values.yaml
+results:
+  - policy: add-default-resources
+    rule: add-default-requests
+    resource: nginx-demo
+    patchedResource: patchedResource.yaml
+    kind: Pod
+    result: pass
+
```

Comment:
@oshi36 please also add at least one test case that shows the result of `skip`. Since this policy uses add-if-not-present anchors, we want to test to ensure that if these fields are present, the mutation is _not_ performed.

Reply:
Yes, added test case for skip result and working as expected.